meshtastic/interfaces/ble/gating.py

```
@@ -331,6 +347,55 @@ def _prune_stale_unowned_claim_locked(key: str) -> bool:
+            "Pruning stale provisional claim for %s (marked_at=%s, threshold=%.1fs).",
+            key,
+            marked_at,
+            BLEConfig.CONNECTION_GATE_UNOWNED_STALE_SECONDS,
+        )
+        _remove_connecting_record_locked(key)
+        _cleanup_addr_lock(key)
+        return True
+    return False
+
+
+def _mark_connecting(addr: str | None, owner: Any | None = None) -> None:
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `owner`

(ANN401)